packages/shared/src/components/modals/BasePostModal.tsx

```diff
@@ -39,9 +42,28 @@ function BasePostModal({
   ...props
 }: BasePostModalProps): ReactElement {
   const { usePostReferrer } = usePostReferrerContext();
+  const { logEvent } = useLogContext();
+  const [scrollNode, setScrollNode] = useState(null);
 
   usePostReferrer({ post });
 
+  const onScroll = useCallback(
+    (event: Event) => {
+      const targetElement = event.target as HTMLElement;
+      logEvent({
+        event_name: LogEvent.PageScroll,
+        target_id: TargetType.Post,
```

Comment:
Thinking about it, let's do `target_type` as Post and target_id as the post id please. Also in the page itself. Sorry for the last minute change